xc/common/utils/prjxray_create_synth_tiles.py

```diff
@@ -57,8 +160,14 @@ def main():
     synth_tiles = {}
     synth_tiles['tiles'] = {}
 
-    with open(args.roi) as f:
-        j = json.load(f)
+    if args.roi:
```

Comment:
Again, if the logic between --roi and --partition_region are the same, why do we need a new flag?

Reply:
I think this flag is still necessary for the distinction at line 209